Don't include the -C option in the help text, since it's gone from the man page.

--- krb5-1.2/src/clients/ksu/main.c.ksu-man	Fri Jun 23 19:45:06 2000
+++ krb5-1.2/src/clients/ksu/main.c	Mon Jun 26 10:36:17 2000
@@ -60,7 +60,7 @@
    ill specified arguments to commands */        
 
 void usage (){
-	fprintf(stderr, "Usage: %s [target user] [-n principal] [-c source cachename] [-C target cachename] [-k] [-D] [-r time] [-pf] [-l lifetime] [-zZ] [-q] [-e command [args... ] ] [-a [args... ] ]\n", prog_name);
+	fprintf(stderr, "Usage: %s [target user] [-n principal] [-c source cachename] [-k] [-D] [-r time] [-pf] [-l lifetime] [-zZ] [-q] [-e command [args... ] ] [-a [args... ] ]\n", prog_name);
 }
 
 /* for Ultrix and friends ... */
